packages/node_modules/@node-red/editor-client/locales/fr/editor.json

```
@@ -0,0 +1,1211 @@
+      "password": "Mot de passe",
+      "ssh-key": "Clé SSH",
+      "passphrase": "Phrase de mot de passe",
+      "desc2": "Avant de pouvoir cloner un référentiel sur ssh, vous devez ajouter une clé SSH pour y accéder.",
+      "add-ssh-key": "Ajouter une clé ssh",
+      "credentials-encryption-key": "Clé de chiffrement des identifiants",
+      "already-exists-2": "existe déjà",
+      "git-error": "erreur git",
+      "con-failed": "La connexion a échoué",
+      "not-git": "Ce n'est pas un dépôt git",
+      "no-resource": "Référentiel introuvable",
+      "cant-get-ssh-key-path": "Erreur! Impossible d'obtenir le chemin de la clé SSH sélectionnée.",
+      "unexpected_error": "Erreur inattendue",
+      "clearContext": "Effacer le contexte lors du changement de projet"
+    },
+    "delete": {
+      "confirm": "Voulez-vous vraiment supprimer ce projet ?"
+    },
+    "create-project-list": {
+      "search": "rechercher vos projets",
+      "current": "actuel"
+    },
+    "require-clean": {
+      "confirm": "<p>Vous avez des modifications non déployées qui seront perdues.</p><p>Voulez-vous continuer ?</p>"
+    },
+    "send-req": {
+      "auth-req": "Authentification requise pour le référentiel",
+      "username": "Nom d'utilisateur",
+      "password": "Mot de passe",
+      "passphrase": "Phrase de mot de passe",
+      "retry": "Recommencer",
+      "update-failed": "La mise à jour a échoué",
+      "unhandled": "Code d'erreur non géré",
+      "host-key-verify-failed": "<p>La vérification de la clé d'hôte a échoué.</p><p>La clé d'hôte du référentiel n'a pas pu être vérifiée. Veuillez mettre à jour votre fichier <code>known_hosts</code> et réessayer.</p>"
+    },
+    "create-branch-list": {
+      "invalid": "Branche invalide",
+      "create": "Créer une branche",
+      "current": "Actuel"
+    },
+    "create-default-file-set": {
+      "no-active": "Impossible de créer un ensemble de fichiers par défaut sans projet actif",
+      "no-empty": "Impossible de créer un ensemble de fichiers par défaut sur un projet non vide",
+      "git-error": "Erreur Git"
+    },
+    "errors": {
+      "no-username-email": "Votre client Git n'est pas configuré avec un nom d'utilisateur/e-mail.",
+      "unexpected": "Une erreur inattendue est apparue",
+      "code": "Code"
+    }
+  },
+  "editor-tab": {
+    "properties": "Propriétés",
+    "envProperties": "Variables d'environnement",
+    "module": "Propriétés des modules",
+    "description": "Description",
+    "appearance": "Apparence",
+    "preview": "Aperçu de l'interface utilisateur",
+    "defaultValue": "Valeur par défaut"
+  },
+  "tourGuide": {
+    "takeATour": "Aperçu",
+    "start": "Commencer",
+    "next": "Suivant",
+    "welcomeTours": "Visite de bienvenue"
+  },
+  "diagnostics": {
+    "title": "Information système"
+  },
+  "languages": {
+    "de": "Allemand",
+    "en-US": "Anglais",
+    "fr": "Français",
+    "ja": "Japonais",
+    "ko": "Coréen",
```

Hi @FabsMuller,
If I understand correctly you want me to add `"pt-BR": "Portugais brésilien"` in the `fr/editor.json` file?
```suggestion
    "ko": "Coréen",
    "pt-BR": "Portugais brésilien",
```

---

Just Portugais is good enough. Also remember to add _"fr":"Français"_ to each editor.json file of the other languages   of the locales directory.

---

@FabsMuller, I understand your idea of generalizing Portuguese but in this case shouldn't we also change
 the `pt-BR` to `pt-PT`? @knolleary, what's your opinion?

---

We have now merged the Portuguese translations via #3804 

So we do want to add a matching entry to the `fr/editor.json` file. The proposed suggestion above is appropriate I believe.

I will rebase this PR to merge to `dev` for 3.1.-beta.2